science/gmt5/Portfile

```diff
@@ -30,12 +29,12 @@ use_xz              yes
 distname            gmt-${version}
 distfiles           ${distname}-src${extract.suffix}
 
-# stealth update 
+# stealth update
 dist_subdir         ${name}/${version}_1
```

Comment:
You may now remove the `_1` part. Or the full line altogether.

Reply:
I just removed the dist_subdir line and added a "conflicts gmt6" line.